.erb-lint.yml

```diff
@@ -23,3 +23,5 @@ linters:
         suggestion: 'Rename classes that are known to be hidden by the Hush plugin'
   SpaceAroundErbTag:
     enabled: true
+  CommentSyntax:
```

Comment:
This adds a check to prevent incorrect comment tags in ERB (examples borrowed from the erb_lint repository):

```erb
<%# erb comment here %>
<%= # bad erb comment here %>
<%
  # apparently this comment syntax is valid?
%>
<% # very; bad erb comment here %>
```